### DISCUSSION TITLE
management cli?

### DISCUSSION THREAD

**Discussion by obnoxxx** [ohmybash/oh-my-bash#591] [Ideas]
I don't think that oh-my-bash has a cli like oh-my-zsh's `omz` (https://github.com/ohmyzsh/ohmyzsh/blob/master/lib/cli.zsh) 

Wouldn't it be useful to have one?

**Comment by akinomyoga** [comment]
That is a part of #270, but that PR is too big and blocked for the changes unrelated to the `omb` command.

**Reply by obnoxxx** [reply]
Thanks, @akinomyoga ! Great to hear that someone is already looking into the addition of an  `omb` cli!
